webrecorder/wabac.js
# 2.12.0 Release

2.12.0 release includes:

live proxy config fixes:
- strip out current self origin from x-orig-location if included, using a relative location
- support array list and dict list of live proxy origins

dsrules: raise maxbitrate for twitter video to 5000000 to match pywb/browsertrix-crawler, save maxbitrate used if save option provided

multiwacz loading change:
- compute and hash of wacz to collection path (instead of page id)
- attempt to load from same wacz, if available, otherwise look in all waczs

blockloaders: support for non-http fetch (#67)
- rename HttpRangeLoader -> FetchRangeLoader to indicate support beyond http but with fetch
- check if fetch protocol matches current location protocol, if so, use fetchloader
- check if fetch() to custom protocol returns, assume fetch is supported, if fetch throws an exception, assume it's not supported

remotewarcproxy: Fix kiwix loading (fixes #69)
- get actual prefix from config.prefix instead of sourceUrl, which now stores original 'prefix:...' id
- readd 'process/browser' for standalone build
- update webpack 5 to latest

eval improvements (#71):
- add tests ensuring '_eval' and '$eval' are not rewritten
- support eval being global-scope aware
- update eval rewriting to pass '.eval(this, arguments, ...)' to WB_wombat_runEval2 (from wombat 3.3.7) to allow wombat to determine if eval is should be evaluated in local or global scope, (see: webrecorder/wombat#89 for more details)

dependency update: wombat 3.3.7, warcio 1.5.1, latest webpack 5
